documentation/alert-runbook.md

```diff
@@ -25,6 +25,10 @@ Alerts when either the `teaching-vacancies-production` or `teaching-vacancies-wo
 
 Alerts when either the `teaching-vacancies-production` or `teaching-vacancies-worker-production` app reports a crash.
 
+## ProdSlowHttpResponse
+
+Alerts when the `teaching-vacancies-production` app http response time is above 3seconds.
```

Comment:
Can we remove the exact time value? If we need to change the alert later on, we don't want to keep changing the doc.

Reply:
done